wavefront_python_sdk/entities/histogram/sender.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+
+"""
+Interface of Histogram Sender for both clients.
+
+@author Hao Song (songhao@vmware.com)
+"""
+
+
+class WavefrontHistogramSender(object):
+    """Interface of Histogram Sender for both clients."""
+
+    # pylint: disable=too-many-arguments
+
+    def send_distribution(self, name, centroids, histogram_granularities,
+                          timestamp, source, tags):
+        """
+        Send Distribution Data.
+
+        Wavefront Histogram Data format
+        {!M | !H | !D} [<timestamp>] #<count> <mean> [centroids]
+        <histogramName> source=<source> [pointTags]
+        Example: "!M 1533531013 #20 30.0 #10 5.1 request.latency
+                  source=appServer1 region=us-west"
+
+        @param name: Histogram Name
+        @type name: str
+        @param centroids: List of centroids(pairs)
+        @type centroids: list
+        @param histogram_granularities: Histogram Granularities
+        @type histogram_granularities: set
+        @param timestamp: Timestamp
+        @type timestamp: long
+        @param source: Source
+        @type source: str
+        @param tags: Tags
+        @type tags: dict
+        """
+        pass
```

Comment:
May throw a `NotImplementedError` here and elsewhere.

https://docs.python.org/3/library/exceptions.html#NotImplementedError